src/actions.js

```diff
@@ -193,7 +193,11 @@ export const types = {
   WALLETCONNECT_NEW_NANOCONTRACT_RETRY: 'WALLETCONNECT_NEW_NANOCONTRACT_RETRY',
   WALLETCONNECT_NEW_NANOCONTRACT_RETRY_DISMISS: 'WALLETCONNECT_NEW_NANOCONTRACT_RETRY_DISMISS',
   SHOW_SIGN_MESSAGE_REQUEST_MODAL: 'SHOW_SIGN_MESSAGE_REQUEST_MODAL',
-  SHOW_NANO_CONTRACT_SEND_TX_MODAL: 'SHOW_NANO_CONTRACT_SEND_TX_MODAL'
+  SHOW_NANO_CONTRACT_SEND_TX_MODAL: 'SHOW_NANO_CONTRACT_SEND_TX_MODAL',
+  SHOW_CREATE_TOKEN_REQUEST_MODAL: 'SHOW_CREATE_TOKEN_REQUEST_MODAL',
```

Comment:
[note] I have proposed a taxonomy for actions: ```<subject>_<action>_<result>```
See at https://github.com/HathorNetwork/hathor-wallet-mobile/issues/334

I would rename this action to:
- `NANOCONTRACT-SEND-TX-MODAL_SHOW`: being `NANOCONTRACT-SEND-TX-MODAL` the subject and `SHOW` the action, or
- `NANOCONTRACT-SENDTXMODAL_SHOW`, or even
- `NCSENDTXMODAL_SHOW`

This may not be the best taxonomy, but better have one then not. Also, feel free to amend the taxonomy proposition and make it better.

Reply:
I will open an issue to refactor this in another PR because I would have to refactor the other actions here